install-mkl-macos.sh

```diff
@@ -0,0 +1,22 @@
+MACOS_BASEKIT_URL=$1
```

Comment:
I think we can make this script generic like `install-intel-windows.bat`, so we can pass the URL and components as args and call it for both basekit and hpckit?